tutorial/docker-compose.yml

```diff
@@ -12,20 +12,9 @@ networks:
 services:
 
   alarms-service:
-    image: iotechsys/alarms-service:0.2 
-    hostname: alarms-service
-    container_name: alarms-service 
     ports:
-      - 8084:8084
```

Comment:
OI!!! this is the graphql port... hands off ;P

Reply:
this docker compose is an override for the edgecentral docker compose. See https://github.com/IOTechSystems/central-cli/blob/c66f6a31bc6405876ed893e6c97d6539f96fc2ad/scripts/composefiles/docker-compose.yml#L328-L345  - the port will remain open